de.metas.adempiere.adempiere/client/src/main/java-legacy/org/compiere/apps/wf/WFPanel.java

```diff
@@ -97,7 +100,7 @@ public void setSimpleWorkflowWindow(boolean isSimpleWorkflowWindow)
 
 	/** Workflow WhereClause : General, Document Process, Document Value */
 	private static final String WORKFLOW_WhereClause = "WorkflowType IN ("
-			+ DB.TO_STRING(MWorkflow.WORKFLOWTYPE_General)
+			+ DB.TO_STRING(X_AD_Workflow.WORKFLOWTYPE_Allgemein)
```

Comment:
Please update the workflowType AD_Ref_List to make it (I think the column is `ValueName`) "General", nocht "Allgemein" etc..i.e. the english version. The german one crept in years ago and keeps popping up here and there..